crates/routes/src/feeds.rs

```diff
@@ -173,18 +225,12 @@ async fn get_feed(
   )
 }
 
-fn get_sort_type(info: web::Query<Params>) -> Result<SortType, ParseError> {
-  let sort_query = info
-    .sort
-    .clone()
-    .unwrap_or_else(|| SortType::Hot.to_string());
-  SortType::from_str(&sort_query)
-}
-
 #[tracing::instrument(skip_all)]
 async fn get_feed_user(
   pool: &DbPool,
   sort_type: &SortType,
+  limit: &i64,
```

Comment:
Surprised clippy isn't saying these are unecessary. It passed the CI tho anyway.